packages/lesswrong/components/sequences/ChaptersItem.tsx

```diff
@@ -73,7 +73,7 @@ const ChaptersItem = ({ chapter, canEdit, classes }: {
           })}
         </AnalyticsContext>
       </div>
-      {!chapter.title && canEdit && <SectionFooter>{editButton}</SectionFooter>}
+      {canEdit && <SectionFooter>{editButton}</SectionFooter>}
```

Comment:
I don't understand why we had this extra condition here - maybe @jimrandomh or @Raemon knows? I was asked to edit a chapter description and I just made this change locally and was able to edit it, so it seems fine to keep?